tcf_website/templates/reviews/edit_review.html

```diff
@@ -0,0 +1,30 @@
+{% extends "base/base.html" %}
+{% load static %}
+
+{% block title %}Edit a Review - theCourseForum{% endblock %}
+
+{% block styles %}
+<link rel="stylesheet" href="{% static 'reviews/new_review.css' %}" />
+{% endblock %}
+
+{% block content %}
+<div class="new-review container text-left mx-auto">
+  <h1 class="h2 mt-2 mb-4">Edit a review</h2>
```

Comment:
Not sure if this is significant, but there's an `<h1>` opening tag and an `</h2>` closing tag here.

Reply:
Fixed in ff3e5e44011622b06df1adf3827d358a091d404a